### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PavlovaOlena**
[DEMO LINK](https://PavlovaOlena.github.io/layout_stars/)
[TEST REPORT LINK](https://PavlovaOlena.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
```
Comment (polosanya) [inline src/index.html:12]:
```suggestion
    <div class="stars stars--0">
```
Don't leave empty lines between parent and child elements

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,54 @@
...
+
+    <div class="stars stars--5">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    
   </body>
```
Comment (polosanya) [inline src/index.html:60]:
```suggestion
    </div>
  </body>
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+* {
+  margin: 0;
+}
```
Comment (polosanya) [inline src/style.css:3]: Don't reset margin using * selector. Better specify the exact tags you want to reset

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost perfect!

**Review by anstsot — APPROVED** [review]
Great job, approved!

**Pull request closed without merging**